test/test.sh

```diff
@@ -166,7 +178,7 @@ echo "$S3_COMMAND setacl $TEST_BUCKET filename=acl"
 $S3_COMMAND setacl $TEST_BUCKET filename=acl
 failures=$(($failures + (($? == 0) ? 0 : 1)))
 
-# Test to make sure that it worked
+# Test to make sure that it worked, the failures will be added when using ceph s3
```

Comment:
libs3 does not claim compatibility with ceph s3 so not sure this comment is necessary